libs/langchain/tests/unit_tests/llms/test_imports.py

```diff
@@ -0,0 +1,8 @@
+from langchain import llms
+
+
+def test_all_imports() -> None:
+    """Simple test to make sure all things can be imported."""
+    for cls in llms.__all__:
+        exec(f"from langchain.llms import {cls}")
+        # TODO: assert all is of type BaseLLM
```

Comment:
lol

We could also use importlib for this, but given this is a test don't care either way

Reply:
I don't think we should use exec -- automated tools aren't gonna exercise discretion about where `exec` is located